prune.go

```diff
@@ -134,8 +134,9 @@ func maybePrune(db *mgo.Database, txnsName string, pruneOpts PruneOptions) error
 	}
 
 	stats, err := CleanAndPrune(CleanAndPruneArgs{
-		Txns:      txns,
-		TxnsCount: txnsCount,
+		Txns:       txns,
+		TxnsCount:  txnsCount,
+		NewestTime: pruneOpts.NewestTime,
```

Comment:
Thinking through this, it has a chance to cause txns to keep growing. At least, if # txns are increasing, each time we evaluate we'll leave more txns behind, delaying when the next prune actually triggers, and then leaving behind more txns which delays the next prune.

I think just basing it on a concrete time is a better backstop.